docs/items/index.md

```diff
@@ -10,23 +10,66 @@ Creating an Item
-|:---------------:|:----------------------------------------------|
-| `tab`           | Sets which `CreativeModeTab` this item is under. Must be called if this item is meant to be shown on the creative menu. Vanilla tabs can be found in the class `CreativeModeTab`. |
-| `durability`    | Sets the maximum damage value for this item. If it is over `0`, two item properties "damaged" and "damage" are added. |
-| `stacksTo`      | Sets the maximum stack size. You cannot have an item that is both damageable and stackable. |
-| `setNoRepair`   | Makes this item impossible to repair, even if it is damageable. |
-| `craftRemainder`| Sets this item's container item, the way that lava buckets give you back an empty bucket when they are used. |
+|      Method        |                  Description                  |
+|:------------------:|:----------------------------------------------|
+| `requiredFeatures` | Sets the required `FeatureFlag`s needed to see this item in the `CreativeModeTab` it is added to. |
+| `durability`       | Sets the maximum damage value for this item. If it is over `0`, two item properties "damaged" and "damage" are added. |
+| `stacksTo`         | Sets the maximum stack size. You cannot have an item that is both damageable and stackable. |
+| `setNoRepair`      | Makes this item impossible to repair, even if it is damageable. |
+| `craftRemainder`   | Sets this item's container item, the way that lava buckets give you back an empty bucket when they are used. |
 
 The above methods are chainable, meaning they `return this` to facilitate calling them in series.
 
 ### Advanced Items
 
 Setting the properties of an item as above only works for simple items. If you want more complicated items, you should subclass `Item` and override its methods.
 
+## `CreativeModeTabEvent`
+
+An item can be added to a `CreativeModeTab` via `CreativeModeTabEvent$BuildContents` on the [mod event bus][modbus]. An item(s) can be added without any additional configurations via `#accept`.
+
+```java
+// Registered on the MOD event bus
+// Assume we have RegistryObject<Item> and RegistryObject<Block> called ITEM and BLOCK
+@SubscribeEvent
+public void buildContents(CreativeModeTabEvent.BuildContents event) {
+  // Add to ingredients tab
+  if (event.getTab() == CreativeModeTabs.INGREDIENTS) {
+    event.accept(ITEM);
+    event.accept(BLOCK); // Takes in an ItemLike, assumes block has registered item
+  }
+}
+```
+
+You can also enable or disable items being added through a `FeatureFlag` in the `FeatureFlagSet` or a boolean determining whether the player has permissions to see operator creative tabs.
+
+### Custom Creative Tabs
+
+A custom `CreativeModeTab` can be created via `CreativeModeTabEvent$Register#registerCreativeModeTab` on the [mod event bus][modbus]. This takes in the name of the tab and a consumer of the builder. In addition, a list of `ResourceLocation`s or `CreativeModeTab`s can be provided to determine where this tab should be located.
```

Comment:
In my opinion, I think an example on one of the methods is fine since this would be passing in two lists with resource locations, something everybody should know how to construct and get the correct method signature within the event. But, I'll leave it up to your discretion on which method should be showcased in the docs.

Reply:
Hmmm. Given that most mods are likely to append a tab to the end instead of ordering based on other tabs, we can omit the example then.